lightningd/json.c

```diff
@@ -299,6 +299,11 @@ void json_add_bool(struct json_stream *result, const char *fieldname, bool value
 	json_add_member(result, fieldname, value ? "true" : "false");
 }
 
+void json_add_null(struct json_stream *stream, const char *fieldname)
+{
+	json_add_member(stream, fieldname, "null");
+}
```

Comment:
This commit should precede the prior one, but moreover: I think the usage here is wrong.  If a field doesn't exist it shouldn't exist, not be set to 'null'.  That's how we handle every other optional field.